Formula/bareos-fd.rb

```diff
@@ -0,0 +1,70 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/16.2.7.tar.gz"
+  sha256 "0b8dbe2e17b3eda470c30d4fe6af92e6f7b668fd1cfc1b045becdc8c8b6a767f"
+
+  depends_on "readline"
+  depends_on "openssl"
+  depends_on :python => :build
```

Comment:
It can't use system `python` to build?

Reply:
Not sure. At least is compiles a C library against Python (to enable Bareos Python plugins), so it does a
#include <Python.h>

However, the plugin itself is untested on MacOS.

Reply:
test, shmest

Reply:
Just drop the dep for now & see what happens 🤷‍♂️. Very few things specifically require Homebrew's Python.